common-items.lic

```diff
@@ -91,6 +91,39 @@ module DRCI
     result =~ /You tap|You thump/
   end
 
+  def count_item_parts(item)
```

Comment:
What is the benefit to moving this from `restock` to here, rather than updating `restock` to handle the functionality you need?

Reply:
The problem is that there's a lot of logic involved in deciding which items we need, e.g. we don't need to restock taffelberries if we know GG, and we don't need wine if we're blacklisting that ritual, etc.  It would make sense to make the core behavior of restock into a script that could be called with a "shopping list", but that was beyond my level of ambition here.

Reply:
It sounds like we need to make restock more dynamic, but that's a PR for another day